### PR TITLE
GM: FPv2 logging

### PR DESCRIPTION
**Pull request by sshane**

### CODE REVIEW

File: selfdrive/car/gm/values.py [hunk 1]
```diff
@@ -141,6 +143,35 @@ class CanBus:
   LOOPBACK = 128
   DROPPED = 192
 
+
+GM_VIN_REQUEST = b'\x1a\x90'
+GM_SOFTWARE_MODULE_1_REQUEST = b'\x1a\xc1'
+GM_SOFTWARE_MODULE_2_REQUEST = b'\x1a\xc2'
+GM_SOFTWARE_MODULE_3_REQUEST = b'\x1a\xc3'
+GM_END_MODEL_PART_NUMBER_REQUEST = b'\x1a\xcb'
+GM_BASE_MODEL_PART_NUMBER_REQUEST = b'\x1a\xcc'
```
Comment (sshane) [inline selfdrive/car/gm/values.py:152]: TODO: we may be able to query multiple ids at once

Comment (sshane) [reply selfdrive/car/gm/values.py:152]: Nope GM's implementation of read data by id only supports one id per request

File: selfdrive/car/gm/values.py [hunk 2]
```diff
@@ -141,6 +143,40 @@ class CanBus:
...
+]
+
+FW_QUERY_CONFIG = FwQueryConfig(
+  requests=[
+    Request(
+      [StdQueries.SHORT_TESTER_PRESENT_REQUEST] + GM_FW_REQUESTS,
+      [StdQueries.SHORT_TESTER_PRESENT_RESPONSE] + [GM_FW_RESPONSE] * len(GM_FW_REQUESTS),
+      rx_offset=0x400,
+      bus=0,
+    ),
```
Comment (sshane) [inline selfdrive/car/gm/values.py:171]:
```suggestion
    Request(
      [StdQueries.SHORT_TESTER_PRESENT_REQUEST] + GM_FW_REQUESTS,
      [StdQueries.SHORT_TESTER_PRESENT_RESPONSE] + [GM_FW_RESPONSE] * len(GM_FW_REQUESTS),
      rx_offset=0x400,
      bus=0,
    ),
    Request(
      [StdQueries.SHORT_TESTER_PRESENT_REQUEST] + GM_FW_REQUESTS,
      [StdQueries.SHORT_TESTER_PRESENT_RESPONSE] + [GM_FW_RESPONSE] * len(GM_FW_REQUESTS),
      rx_offset=0x400,
      bus=1,
    ),
    Request(
      [StdQueries.SHORT_TESTER_PRESENT_REQUEST] + GM_FW_REQUESTS,
      [StdQueries.SHORT_TESTER_PRESENT_RESPONSE] + [GM_FW_RESPONSE] * len(GM_FW_REQUESTS),
      rx_offset=0x400,
      bus=1,
      obd_multiplexing=False,
    ),
```

Comment (sshane) [reply selfdrive/car/gm/values.py:171]: add more queries

**Comment by Verylukyguy** [comment]
I loaded this branch and got "CAN Error: Check Connections" on my Comma 3 and several dash warnings in my cluster.

**Comment by sshane** [comment]
This doesn't work yet. See issue: https://github.com/commaai/openpilot/pull/27929

**Pull request closed without merging**